website/source/docs/provisioning/ansible_common.html.md

```diff
@@ -146,3 +185,16 @@ Some of these options are for advanced usage only and should not be used unless
     Examples: `true` (equivalent to `v`), `-vvv` (equivalent to `vvv`), `vvvv`.
 
     Note that when the `verbose` option is enabled, the `ansible-playbook` command used by Vagrant will be displayed.
+
+- `version` (string) - The expected Ansible version.
+
+    This option is disabled by default.
+
+    When an Ansible version is defined (e.g. `"2.1.6.0"`), the Ansible provisioner will be executed only if Ansible is installed at the requested version.
+
+    When this option is set to `"latest"`, no version check is applied.
+
+    <div class="alert alert-info">
+      <strong>Tip:</strong>
+      With the `ansible_local` provisioner, it is currently possible to use this option to specify which version of Ansible must be automatically installed, but <strong>only</strong> in combination with the [**`install_mode`**](ansible_local.html#install_mode)  set to <strong>`:pip`</strong>.
```

Comment:
@chrisroberts I've just noticed that it is totally okay for `middleman` rendering, to link HTML pages with the same directory without the whole path (e.g. `ansible_local.html#install_mode` instead of `/docs/provisioning/ansible_local.html#install_mode`).
If I'm correct, do you agree if I replace all these links (at least within the `/docs/provisioning/ansible*.html` scope) ?

Reply:
@gildegoma All the documentation currently uses the full path. Would prefer to keep things using the full path for consistency throughout the documentation pages.

Reply:
Agreed! I'll fix that one soon. Thanks for the quick reply!!